blueocean-dashboard/src/main/js/components/Branches.jsx

```diff
@@ -86,12 +91,20 @@ export default class Branches extends Component {
                     />
                 </CellLink>
                 <td className="actions">
+
                     <RunButton
                       className="icon-button"
                       runnable={branch}
                       latestRun={branch.latestRun}
                       onNavigation={openRunDetails}
                     />
+
+                    <div className="history-button-component">
+                        <Link to={historyButtonUrl} className="materials-icons">
```

Comment:
it's not terrible to have named divs in some cases (I know I often look for them, saves a lot of time in debugging and testing - which isn't a good reason, but I believe its not terrible practice as well).